src/components/CreateSpotAndPersona/SpotAndPersonaProps.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
we use FC instead of FunctionComponent  everywhere, let's stick to that

Reply:
done